velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -156,7 +161,17 @@ class PlanBuilder {
           std::string,
           std::shared_ptr<connector::ColumnHandle>>& assignments);
 
+  /// Add a TableScanNode to scan a TPC-H table.
+  ///
+  /// @param tpchTableHandle The handle that specifies the target TPC-H table
+  /// and scale factor.
+  /// @param columnNames The columns to be returned from that table.
+  PlanBuilder& tableScan(
+      const std::shared_ptr<connector::tpch::TpchTableHandle>& tpchTableHandle,
+      std::vector<std::string>&& columnNames);
```

Comment:
Should the "columnNames" parameter be const && ?

Reply:
You mean const &? The intent here is to move it in to this this function since the ROW() function expects an rvalue ref (so we can avoid useless copies)

Reply:
Alright. Thanks.